README.md

```diff
@@ -70,6 +92,11 @@ However, coverage insights are arguably even more pertinent for complex and mono
 * Execution time is effectively proportional to the product of the number test cases and the size of the SQL code base under test.
 * Project currently includes only a small set of gene matchers and possible mutations, this limits the scope of coverage measurement.
 * The responsibility for unit test execution and mutation testing is currently conflated.
+* If we want to use JUnit5, we have to change every `@Test` annotation in the test class with `@MutantSwarmTest`. Ideally, we would like to simply place the
+  `@MutantSwarmTest` annotation at the top of the code and leave the `@Test` annotations unchanged. [Link to this issue in stackoverflow](https://stackoverflow.com/questions/64872557/annotate-a-full-class-with-an-extension-while-intercepting-all-the-tests-and-in).
+* If you run maven with the JUnit5 extension, the build will fail. This is because some of the tests with the mutations shouldn't pass, and since
```

Comment:
Worth mentioning this in the JUnit 5 section?

Reply:
I think so too. The problem isn't directly tied to Maven, the JUnit tests themselves show up as failed, that's the root problem so we should describe it like that.